pages/glossary/can-be-targeted-by-pointer-event.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Can be target by a pointer event
+key: can-be-targeted-by-pointer-event
+unambiguous: true
+objective: true
+input_aspects:
+  - CSS styling
+  - DOM tree
+---
+
+An element _can be targeted by pointer events_ when all the following conditions are true:
+
+- the element is a [semantic `widget`][semantic role]; and
+- the element is [focusable][]; and
+- the element's [border box][] intersects the viewport; and
+- the element's [border box][] is not entirely covered by the [border boxes][] of elements with greater `z-index` and `pointer-events: auto` CSS properties.
```

Comment:
Want to consider also open dialogs?
https://html.spec.whatwg.org/multipage/interactive-elements.html#attr-dialog-open

Reply:
I'm not sure exactly what you mean 🤔 

Is it about elements "behind" the modal being blocked by it when it's opened (even if they are visible)?
I think that this is covered because in that case, their "clickable area" would be empty (the blocked elements are never topmost target, until the modal is closed and stops blocking them).